src/server/mappers/UrlMapper.ts

```diff
@@ -14,16 +14,27 @@ export class UrlMapper implements Mapper<StorableUrl, UrlType> {
     const { UrlClicks: urlClicks } = urlType
     if (!urlClicks || !Number.isInteger(urlClicks.clicks))
       throw new Error('UrlClicks object not populated.')
+
+    const tagStrings: string[] = []
+    const { tags } = urlType
+    if (tags) {
+      for (let i = 0; i < tags.length; i += 1) {
+        tagStrings.push(tags[i].tagString)
```

Comment:
should we use the `TagMapper.persistenceToDto` function here?

Reply:
it will just change tags[i].tagString to tagMapper.persistenceToDTO(tags[i]). Do we really want to do that?

Reply:
I was thinking about `tags.map((tag) => tagMapper.persistenceToDTO(tag))`

Reply:
makes sense, done